api/napi/cover.md

```diff
@@ -0,0 +1,166 @@
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+# Slint-node (Beta)
+
+[![npm](https://img.shields.io/npm/v/slint-ui)](https://www.npmjs.com/package/slint-ui)
+
+[Slint](https://slint.dev/) is a UI toolkit that supports different programming languages.
+Slint-node is the integration with node.
+
+To get started you can use the [Walk-through tutorial](https://slint.dev/docs/tutorial/node).
+We also have a [Getting Started Template](https://github.com/slint-ui/slint-nodejs-template) repository with
+the code of a minimal application using Slint that can be used as a starting point to your program.
+
+**Warning: Beta**
+Slint-node is still in the early stages of development: APIs will change and important features are still being developed.
+
+## Slint Language Manual
+
+The [Slint language manual](../slint) covers the Slint UI description language
+in detail.
+
+## Installing Slint
+
+Slint is available via NPM, so you can install by running the following command:
+
+```sh
+npm install slint-ui
+```
+
+### Dependencies
+
+You need to install the following components:
+
+  * **[Node.js](https://nodejs.org/download/release/v16.19.1/)** (v16. Newer versions currently not supported: [#961](https://github.com/slint-ui/slint/issues/961))
+  * **[npm](https://www.npmjs.com/)**
+  * **[Rust compiler](https://www.rust-lang.org/tools/install)** (1.70 or newer)
+
+You will also need a few more dependencies, see <https://github.com/slint-ui/slint/blob/master/docs/building.md#prerequisites>
+
+## Using Slint
+
+To initialize the API, you first need to import the `slint-ui` module in our code:
+
+```js
+import * as slint from "slint-ui";
+```
+
+As next you can load a slint file with:
+
+```js
+let ui = slint.loadFile("ui/main.slint");
+```
+
+Combining these two steps leads us to the obligatory "Hello World" example:
+
+```js
+import * as slint from "slint-ui";
+let ui = slint.loadFile(".ui/main.slint");
+let main = new ui.Main();
+main.run();
+```
+
+See [/examples/todo/node](https://github.com/slint-ui/slint/tree/master/examples/todo/node) for a full example.
+
+## API Overview
+
+### Instantiating a component
+
+The exported component is exposed as a type constructor. The type constructor takes as parameter
+an object which allow to initialize the value of public properties or callbacks.
+
+```js
+import * as slint from "slint-ui";
+// In this example, the main.slint file exports a module which
+// has a counter property and a clicked callback
+let ui = slint.loadFile("ui/main.slint");
+let component = new ui.MainWindow({
+    counter: 42,
+    clicked: function() { console.log("hello"); }
+});
+```
+
+### Accessing a property
+
+Properties are exposed as properties on the component instance
+
+```js
+component.counter = 42;
+console.log(component.counter);
+```
+
+### Callbacks
+
+The callbacks are also exposed as property that have a setHandler function, and that can can be called.
```

Comment:
```suggestion
The callbacks are also exposed as property that has a setHandler function, and that can be called as a function.
```

Please complete the example with .slint code, so that the example is self-contained.